codegen/src/main/java/io/improbable/keanu/codegen/python/ParamStringProcessor.java

```diff
@@ -0,0 +1,28 @@
+package io.improbable.keanu.codegen.python;
+
+import com.sun.javadoc.ConstructorDoc;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.google.common.base.CaseFormat.LOWER_UNDERSCORE;
+import static com.google.common.base.CaseFormat.UPPER_CAMEL;
+
+class ParamStringProcessor {
```

Comment:
I think this class would benefit from some tests - especially considering various edge cases around how the javadoc might be formatted.